solutions/usaco-1131.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: usaco-1131
+source: USACO Bronze 2021 US Open
+title: Acowdemia I
+author: Juheon Rhee
+---
+
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_prob1_bronze_open21.html)
+
+## Implementation
+
```

Comment:
(optional)

could you add the time complexity?

Reply:
yep